AU2/plugins/CorePlugin.py

```diff
@@ -168,8 +170,19 @@ def __init__(self):
             )
         ]
 
-        # str -> HookedExport
-        self.hooks = {}
+        self.hooks = {
+            "hide_assassins": self.identifier + "_hide_assassins",
+        }
+
+        self.hooked_exports = [
+            HookedExport(

```

Comment:
You can make this an API call exposed directly in `AbstractPlugin` with a default
"Do nothing" implementation, like the similar `on_assassin_create`. `HookedExport`s will work here but they're a little clunkier and mostly intended for plugins that can be enabled or disabled at will (hence not justifying a dedicated API call for their functionality).

Reply:
The reason I did this as a hook is that, I was thinking that once #63 is implemented there wouldn't be much reason to use this export and we'd want to move it into a debug plugin à la #36. I was also thinking that the only reason it needs a hook at all is for `UIConfigPlugin`, so didn't justify its own call.

But come to think of it, we might want also to a, for example, have `TargetingPlugin` give a warning when hiding live assassins that hidden assassins will still be included in the targeting tree. So maybe it does justify an API call?